Firestore/Example/Tests/Local/FSTLevelDBMigrationsTests.mm

```diff
@@ -60,43 +61,54 @@ - (void)tearDown {
 - (void)testAddsTargetGlobal {
   FSTPBTargetGlobal *metadata = [FSTLevelDBQueryCache readTargetMetadataFromDB:_db];
   XCTAssertNil(metadata, @"Not expecting metadata yet, we should have an empty db");
-  [FSTLevelDBMigrations runMigrationsOnDB:_db];
+  LevelDBTransaction transaction(_db, [FSTLevelDB standardReadOptions],
```

Comment:
As noted in the underlying PR, let's just make these default arguments on `LevelDBTransaction`'s constructor.

Reply:
Done.